lib/id3c/cli/command/etl/redcap_det.py

```diff
@@ -122,12 +122,37 @@ def decorated(*args, db: DatabaseSession, log_output: bool, **kwargs):
                             mark_skipped(db, det.id, etl_id)
                             continue
 
-                        redcap_record = get_redcap_record_from_det(det.document, raw_coded_values)
+                        # Check if this is record has an older DET
+                        # Skip older DET in favor of the latest DET
+                        elif latest_complete_dets.get(record_id):
+                            old_det = latest_complete_dets[record_id]
+                            LOG.debug(f"Skipping older REDCap DET {old_det.id}")
+                            mark_skipped(db, old_det.id, etl_id)
+
+                        latest_complete_dets.update({record_id: det})
```

Comment:
Since we're working on a PR related to optimization, I wanted to point out that since we're only updating one key-value, python's "regular" (don't know a better word here) way of updating a single value in a dict is faster than using `.update()`.

e.g.
```
import timeit

timeit.timeit("foo.update({'a': 4})", setup="foo = {'a': 3}")
```
Runs in ~0.18s, whereas
```
timeit.timeit("foo['a'] = 4", setup="foo = {'a': 3}")
```
runs in ~0.06s (about 1/3 that of `.update()`).

Reply:
You are absolutely right. I was probably overthinking things when I wrote this. 😄